scripts/utils/default_yargs.js

```diff
@@ -0,0 +1,12 @@
+module.exports = require("yargs")
+  .version(false)
+  .strict()
```

Comment:
oh yeah, strict is good to have!

Reply:
What does strict do/mean?

Reply:
Strict means that the script fails if any of the arguments used during its invocation is not explicitly defined by us in yargs.